package.json

```diff
@@ -123,6 +124,7 @@
     "prettier": "^3.3.1",
     "rimraf": "^5.0.7",
     "semantic-release": "^24.0.0",
+    "semver": "^7.6.2",
```

Comment:
This is not good. While it's true that `semver` knows better how to compare versions, it's not a good idea to make the primary package depend on something required for another package and feature. I'm not even sure that we can place it in `devDependencies` as we are running it on the client side. (while placing in `dependecies` is not allowed for sure :) )